apps/docs/docs/tutorials/how-to-deep-link.md

```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
 ---
 # Deep link to Portal Bridge
 
+> Note: this feature could work differently between the different bridged experiences
+> that currently portal bridge offers. The documentation presented here works on https://portalbridge.com
```

Comment:
Rewrite this as follows: Note: This feature could work differently between the different bridge versions than what Portal Bridge currently offers. The documentation presented here works on https://portalbridge.com